pyproject.toml

```diff
@@ -0,0 +1,53 @@
+[tool.poetry]
+name = "up42-py"
+version = "0.32.0"
+description = ""
```

Comment:
consider adding description, authors, and licence. 
You can take a look [here](https://github.com/up42/up42-py/blob/configure_poetry/pyproject.toml) for reference